run_tests.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
-from flake8_builtins import BuiltinsChecker
-
 import ast
 import unittest
 
+from flake8_builtins import BuiltinsChecker
+
```

Comment:
@karamanolev yes, I'm aware of it, though, I'm usually working with Plone related code bases, and the Plone community has a different styleguide regarding imports: *sort them alphabetically*.

The use cases are different but I find it pleasantly to not have to think at all about imports, just sort them alphabetically (or let it isort do it for you) and carry on.